pep-9999.rst

```diff
@@ -322,8 +322,9 @@ support the tuple-based version check syntax::
         # This is only necessary when a feature has no Python 3 equivalent.
         # "<= (2, 7)" does not work because e.g. (2, 7, 1) > (2, 7).
 
-Type stubs should avoid checkinging against ``sys.version_info.major``
-directly.
+Type stubs should avoid checking against ``sys.version_info.major``
+directly and should not use comparison operators other than ``<``, ``<=``,
```

Comment:
The comment in the last example (' "<= (2, 7)" does not work because e.g. (2, 7, 1) > (2, 7)') suggests that '<=' (and, by extension, '>'?) should be avoided.

Reply:
Same with `>` really. I am changing it accordingly.